### DISCUSSION TITLE
Current state of meta-rauc-tegra

### DISCUSSION THREAD

**Discussion by hannes09** [rauc/meta-rauc-community#74] [General]
What is the current state of meta-rauc-tegra?  Is it working with the latest realese from meta-tegra (JP 5.1.2) for jetson xavier?

I ran into the following error:

> ERROR: No recipes in default available for:
>   /meta-rauc-community/meta-rauc-tegra/recipes-bsp/u-boot-tegra/u-boot-tegra_%.bbappend

**Comment by leon-anavi** [comment]
Hi @hannes09, is this the same issue as https://github.com/rauc/meta-rauc-community/issues/69 ?

**Reply by hannes09** [reply]
Yes, it is the same error. I tested kirkstone, and it has now the same error.

**Comment by shahzi176** [comment]
Hi, is there any update for Nvidia tegra layer?
